src/parser.ts

```
@@ -367,6 +395,265 @@ export class FileParser {
+            const walk = (node: any) => {
+                if (!node || typeof node !== 'object') return;
+
+                switch (node.kind) {
+                    case 'namespace':
+                        if (node.name) {
+                            keywordSet.add(node.name.toLowerCase());
+                        }
+                        break;
+
+                    case 'class':
+                        const className = getName(node.name);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)